components/SectionExplorer.vue

```diff
@@ -44,8 +44,12 @@ export default {
 }
 </script>
 
-<style lang="stylus" scoped>
+<style lang="stylus">
+.section
+  overflow visible !important
```

Comment:
What's going on here? This is causing an unwanted horizontal scroll.

If you're trying to prevent the top of the graphics being clipped, I'd suggest we just remove the negative top margin and instead remove the bottom padding from the preceding section `.section-testnets`.